understand/observed-size-spectra.qmd

```diff
@@ -94,14 +94,14 @@ The second way to deal with the fact that there are so many more small fish than
 ::: {.callout-note collapse="true"}
 ### Expand for R details
 
-We had decided that we wanted the breaks between the bins to increase with powers of 2. We first create the vector `log_breaks` with the exponents. The `seq()` function creates a vector of numbers starting at `from` and going up to `to` in steps of size `by`. You do not need to give the names of the arguments because they can also be identified by their position. So you could also have written `seq(0, 11, 1)`. Such integer sequences are used so often that there is the even shorter notation `0:11` giving the same sequence.
+We had decided that we wanted the breaks between the bins to be located at powers of 2. We first create the vector `log_breaks` with the exponents. The `seq()` function creates a vector of numbers starting at `from` and going up to `to` in steps of size `by`. You do not need to give the names of the arguments because they can also be identified by their position. So you could also have written `seq(0, 11, 1)`. Such integer sequences are used so often that there is the even shorter notation `0:11` giving the same sequence.
 
-The second line above creates a vector containing the powers of 2 with the exponents we just created in the vector `log_breaks`.
+The second line above creates a vector containing the powers of 2 with the exponents we just created in the vector `log_breaks`. Note how R can perform calculations for all entries of a vector at once, without the need for a loop.
 
 The reason we have put parentheses `( ... )` around the assignments in the above code is because that also leads to the result being displayed whereas the assignment operator without those parentheses around it would not display anything.
 :::
 
-Now we can use these logarithmically increasing bins in the histogram:
```

Comment:
If by "logarithmically increasing" you refer to the size of the bins, then this is incorrect. The size of the bins increases exponentially. The term "logarithmically spaced" is sometimes used as a short hand for "equally spaced on a logarithmic scale". Given that we explained already what our bins are, I think using this shorthand is fine here.